loom/common/assets/assets.h

```diff
@@ -101,6 +101,8 @@ enum LoomAssetType
     LATMesh         = LOOM_FOURCC('M', 'S', 'H', 1),
     LATScript       = LOOM_FOURCC('L', 'O', 'O', 'M'),
     LATBinary       = LOOM_FOURCC('B', 'I', 'N', 1),
+    LATVertexShader = LOOM_FOURCC('V', 'E', 'R', 'T'),
```

Comment:
Sure it could, but if we're gonna do live reloading, this could be useful (not sure though)